specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json

```diff
@@ -0,0 +1,742 @@
+        }
+      },
+      "description": "Restore parameters."
+    },
+    "ResultStatistics": {
+      "type": "object",
+      "properties": {
+        "progress": {
+          "readOnly": true,
+          "type": "number",
+          "multipleOf": 0.01,
+          "description": "Search job completion percentage."
+        },
+        "ingestedRecords": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of rows that were returned by the search job."
+        }
+      },
+      "description": "Search job execution statistics."
+    },
+    "TableProperties": {
+      "type": "object",
+      "properties": {
+        "retentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 730,
+          "description": "The data table data retention in days, between 4 and 730. Setting this property to null will default to the workspace retention."
+        },
+        "totalRetentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4,
+          "maximum": 2555,
+          "description": "The table data total retention in days, between 4 and 2555. Setting this property to null will default to table retention."
+        },
+        "archiveRetentionInDays": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "The table data archive retention in days. Calculated as (totalRetentionInDays-retentionInDays)"
+        },
+        "searchResults": {
+          "$ref": "#/definitions/SearchResults",
+          "description": "Parameters of the search job that initiated this table."
+        },
+        "restoredLogs": {
+          "$ref": "#/definitions/RestoredLogs",
+          "description": "Parameters of the restore operation that initiated this table."
+        },
+        "resultStatistics": {
+          "$ref": "#/definitions/ResultStatistics",
+          "description": "Search job execution statistics."
+        },
+        "plan": {
+          "type": "string",
+          "description": "The table plan.",
+          "enum": [
+            "Basic",
+            "Analytics"
+          ],
+          "x-ms-enum": {
+            "name": "TablePlanEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Basic",
+                "description": "Basic - logs that are adjusted to support high volume / low value verbose logs."
+              },
+              {
+                "value": "Analytics",
+                "description": "Analytics - logs that allow monitoring and analytics."
+              }
+            ]
+          }
+        },
+        "lastPlanModifiedDate": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The timestamp that table plan was last modified (UTC)."
+        },
+        "createDate": {
```

Comment:
>createDate [](http://example.com/codeflow?start=9&length=10)

this will be in systemData and does not need to be duplicated in the property bag.  At best it is duplication, at worst they won't be the same exact timestamp and will look sloppy

Reply:
Taking the example of some other resource, today that information is only provided by the RP, not by ARM. so I miss to see the duplication:
![image](https://user-images.githubusercontent.com/6184843/147729050-1da9adf8-c8fb-4509-a50c-d783c45ccc99.png)

Reply:
I agree, I will fix the duplication and will make sure to propagate that information under system data. btw, is it expected we will generate it explicitly? because I never saw ARM generating it automatically.